peakinvestigator/internal/test_progress.py

```diff
@@ -0,0 +1,37 @@
+## -*- coding: utf-8 -*-
```

Comment:
I get that we were discussing these particular classes, but why are they referred to as Test?